config/routes.rb

```diff
@@ -1,8 +1,11 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  get '', to: 'users#index'
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  get '/users/:id', to: 'users#show'
+
+  get '/users/:user_id/posts', to: 'posts#index'
+
+  get '/users/:user_id/posts/:id', to: 'posts#show'
```

Comment:
### Optional
- [ ] It would be nice to use the resources for nesting our routes instead of manually nesting it as we have currently. Here is how the resources nesting is done
```
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
end

```

Reply:
@addod19 ,

Thanks a lot